gulpfile.js

```diff
@@ -54,5 +55,25 @@ gulp.task('fonts', function() {
         .pipe(gulp.dest(path.join(output_directory, 'fonts')));
 });
 
+// Extracts translatable strings using angular-gettext and saves them in file
+// openslides/locale/angular-gettext/template-en.pot.
+gulp.task('pot', function () {
+    return gulp.src(['openslides/core/static/templates/*.html',
+                     'openslides/users/static/templates/users/*.html',
+                     'openslides/users/static/js/users/*.js'])
```

Comment:
done. I used ``openslides/*/static/templates/*/*.html and openslides/*/static/js/*/*.js`` which works.

@normanjaeckel how can we use translations for plugins? Should each plugin provide there own json files for each language?